### DISCUSSION TITLE
Locked out of account

### DISCUSSION THREAD

**Discussion by mkbctrl** [vercel/community#726] [Help] [answered]
Hey guys,

I need to have two separate accounts, one personal, one company. I have one phone number.

I have deleted the personal account, but it seems your DB still holds it. How can I create new account then?
Is it possible to have two accounts on a single phone number?

Can't start a regular support thread cause I can't get through phone number authorisation. It tells the number is taken, whatever I try. I understand why you have that kind of measures, but in situation like mine it's super annoying 😠

**Comment by mkbctrl** [comment]
Tried the workaround with 10 minute phone numbers, but it looks like you have it covered as well 😅

**Reply by Vadorequest** [reply]
I've opened a ticket from our corporate account to notify the support team. Expect 24-48h delay for an answer. :)

**Reply by Vadorequest** [reply]
@mkbctrl 
![image](https://user-images.githubusercontent.com/3807458/179427620-eaa3320a-3853-49d5-a44e-7f10404a6671.png)

**Comment by JTaylor0196** [comment, marked as answer]
Hi @mkbctrl! You can send an email to support@vercel.com, and someone on the team will be able to help you. Feel free to link this discussion for context, and please be sure to include your phone number in the email!
